src/main/kotlin/com/jetbrains/snakecharm/SnakemakeRuleInheritanceMarkerProvider.kt

```diff
@@ -31,9 +31,11 @@ class SnakemakeRuleInheritanceMarkerProvider : RelatedItemLineMarkerProvider() {
         use: SmkUse,
         result: MutableCollection<in RelatedItemLineMarkerInfo<*>>
     ) {
-        val name = use.getIdentifierLeaf() ?: return
+        val name = (if (use.nameIdentifierIsWildcard()) use.nameIdentifier?.firstChild else use.nameIdentifier) ?: return
```

Comment:
NameIdentifier is `SmkUseNameIdentifier`. Seems it isn't always SmkUseNameIdentifier, sometimes is just Identifier which inconvenient. But in case of wildcards it is `SmkUseNameIdentifier` on runtime. So please introduce proper PSI api instead of `.firstChild`. Also consider moving `use.nameIdentifierIsWildcard()` to `use.nameIdentifier.isWIldcard()`

At the moment name identifier usage for `use` is a bit complicated and not well structured. Likely some PSI cleanup is requied.